src/frontend/src/main/Filters/FiltersBtn.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './filters.css';
+
+export default function FiltersBtn({
+  isDown, isUp, onClick, sortType, children, reverse,
+}) {
+  function prevSort(prev) {
+    if (prev === sortType) {
+      return reverse;
+    }
+    return sortType;
+  }
+  return (
+    <button
+      className={`main-filters__btn ${isDown || isUp ? 'active' : ''} ${isUp ? 'up' : ''}`}
```

Comment:
write out function from `${isDown || isUp ? 'active' : ''} ${isUp ? 'up' : ''}` and reuse it here